megamek/src/megamek/common/Entity.java

```diff
@@ -36,6 +36,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Collectors;
 
+import megamek.client.bot.princess.WeaponFireInfo;
```

Comment:
What is this include for?  I don't know that it should be here.  Generally, we don't want to import things from the client namespace into common (yes, I know GUIPreferences is there.  We should fix that).

Reply:
Don't know how the WeaponFireInfo import got there. It's removed. GuiPreferences is used on lines 650-51 for Climb Mode and 5822 for the same. I left it in there.